src/scripts/install.py

```diff
@@ -63,6 +63,19 @@ def log_level():
 
     return (options, args)
 
+def prepend_destdir(path):
+    """
+    Needed because os.path.join() discards the first path if the
+    second one is absolute, which is usually the case here. Still, we
+    want relative paths to work and leverage the os awareness of
+    os.path.join().
+    """
+    try:
+        destdir = os.environ['DESTDIR']
+    except KeyError as e:
+        destdir = ""
+    return os.path.join(destdir, path.lstrip('/'))
```

Comment:
This is buggy: path=/root/dir DESTDIR unset -> "root/dir"

Additionally we should raise an exception if this is used on Windows (i.e. if DESTDIR is set during install run) because the implementation is far away from being compatible with windows paths. If you have "C:\Users\Someone" as argument in prepend_destdir, you're doomed and risk corrupting data because of writes to wrong paths.